docs/admin/config.rst

```diff
@@ -1204,7 +1232,7 @@ Example:
 STATUS_URL
 ----------
 
-URL where your Weblate instance reports it's status.
+The URL where your Weblate instance reports it's status.
```

Comment:
```suggestion
The URL where your Weblate instance reports it's status.
```
Where they can be found, or where they are sent?

Reply:
Where they can be found. On Hosted Weblate it points to https://status.weblate.org/ which is operated by third party.